chain/chain.go

```diff
@@ -7,17 +7,26 @@ import (
 	"math/big"
 	"os"
 
+	"github.com/hashicorp/go-multierror"
+
+	"github.com/0xPolygon/polygon-edge/helper/common"
 	"github.com/0xPolygon/polygon-edge/helper/hex"
 	"github.com/0xPolygon/polygon-edge/types"
-	"github.com/hashicorp/go-multierror"
 )
 
-var (
+const (
+	// GenesisBaseFee is the initial base fee for EIP-1559 blocks.
+	// Using 1 gwei by default
+	GenesisBaseFee = 1_000_000_000
```

Comment:
WDYT?
```suggestion
	GenesisBaseFee = ethgo.Gwei(1).Uint64()
```

Reply:
It would not be a constant in this case

Reply:
Obviously, but then you don't need the comment above which explains the const value. Code is slightly more readable.

Reply:
Done